src/PolynomialFunction.java

```diff
@@ -0,0 +1,170 @@
+import java.util.regex.*;
+import java.util.Arrays;
+import java.util.*;
+
+public class PolynomialFunction implements ParametricFunction {
+
+    private double[] coefficients;
+    public String functionString;
+    private static final double DEFAULT_START_X = -100.0;
```

Comment:
These default values should be dynamic and represent the left and right edges of the world space (including camera position/scale)

Reply:
Done